mpisppy/extensions/scenario_lpfiles.py

```diff
@@ -0,0 +1,40 @@
+###############################################################################
+# mpi-sppy: MPI-based Stochastic Programming in PYthon
+#
+# Copyright (c) 2024, Lawrence Livermore National Security, LLC, Alliance for
+# Sustainable Energy, LLC, The Regents of the University of California, et al.
+# All rights reserved. Please see the files COPYRIGHT.md and LICENSE.md for
+# full copyright and license information.
+###############################################################################
+""" Extension to write an lp file for each scenario and a json file for
+the nonant structure for each scenario (yes, for two stage problems this
+json file will be the same for all scenarios.)
+"""
+
+import json
+import mpisppy.extensions.extension
+
+
+def lpize(varname):
```

Comment:
Is there a function already in pyomo we can just import to do this?

Reply:
good idea